aot/src/test/resources/com/dylibso/chicory/approvals/approve-all.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Sneaked this in, can remove if requested.

Reply:
I like this. Is there a way to run the tests from Maven without the diffs popping up individually in IntelliJ?

Reply:
Thanks for looking into it, I remove the script in this case.
But I need to add this note somewhere in the docs, otherwise I'm going to forget pretty quickly :sweat_smile:

Reply:
> Is there a way to run the tests from Maven without the diffs popping up individually in IntelliJ?

It was happening to me on a Mac, now on Linux is I don't have this horrific behavior anymore, but I can't recall if I did something about it ...